frontend/test-config.helper.ts

```diff
@@ -0,0 +1,22 @@
+import { TestBed, TestBedStatic } from '@angular/core/testing';
+
+type CompilerOptions = Partial<{
+  providers : any[];
```

Comment:
any is not allowed

Reply:
Is this test-config.helper.ts should exist? I have issue with Jest autocompletion, It's working Jest but not the auto-completion.

Reply:
Usually @types/jest should solve it... of course this has to be added to the tsconfig for testing suite